program/rust/src/tests/test_init_mapping.rs

```diff
@@ -75,7 +76,7 @@ fn test_init_mapping() {
             &[funding_account.clone(), mapping_account.clone()],
             instruction_data
         ),
-        Err(ProgramError::InvalidArgument)
+        Err(OracleError::InvalidFundingAccount.into())
```

Comment:
note on pr structure: you shouldn't have changed these error codes at the same time as the rest of the PR. if you sent a PR with just this error change, we would insta-approve it.